tools/netlist/netlist.cpp

```diff
@@ -232,6 +246,11 @@ int main(int argc, char** argv) {
             return 0;
         }
 
+        if (combLoops == true) {
+            ElementaryCyclesSearch ecs(netlist);
+            ecs.getElementaryCycles();
+            ecs.dumpCyclesList(netlist);
```

Comment:
If you can create a `NetlistPath` from the graph analysis, the path can be reported using slang's diagnostics.

Reply:
I'll take a look at this.

Reply:
Done